pkg/cloud/api/check_test.go

```diff
@@ -258,6 +258,7 @@ func TestCheckSchema(t *testing.T) {
 		wantErr bool
 	}{
 		{name: "ok", t: reflect.TypeOf(&okSt{})},
+		{name: "ok-untyped", t: reflect.TypeOf(&MissingResourceType{})},
```

Comment:
Change to `{name: "PlaceholderType is ok"}`

You should always align the names so people have an easy time following the code:

Here we has "untyped", then the type is "MissingResourceType". I put below that we
should change the name to PlaceholderType. Let's make sure all of the references and
places where we use things use the same kinds of names.